Fedora: Unable to find XShm.h

I'm trying to build on Fedora and getting an error about `XShm.h` being missing. 

    In file included from platform/linux/acpllnx.cpp:23:
    /usr/include/xalleg.h:38:10: fatal error: X11/extensions/XShm.h: No such file or directory
    #include <X11/extensions/XShm.h>
            ^~~~~~~~~~~~~~~~~~~~~~~

I'm not sure what package I should be installing on Fedora to get this library. Any ideas?

I had to install both `libXext-devel` and `libXxf86vm-devel`. I'll update the Fedora compile docs as soon as I get this working.

Now I'm getting:

    /usr/bin/ld: ../Common/libsrc/aastr-0.1.1/AAROT.o: in function `_aa_rotate_bitmap':
    /home/bakers/github/ags/Engine/../Common/libsrc/aastr-0.1.1/AAROT.c:88: undefined reference to `fsin'
    /usr/bin/ld: /home/bakers/github/ags/Engine/../Common/libsrc/aastr-0.1.1/AAROT.c:89: undefined reference to `fcos'
    collect2: error: ld returned 1 exit status

I enable XXX and it gets a lot closer to compiling, but I'm getting a new error about a duplicate function definition now:

    In file included from /usr/include/allegro/base.h:41,
                    from /usr/include/allegro.h:25,
                    from ../Common/util/wgt2allg.h:30,
                    from ../Engine/ac/draw.h:26,
                    from ac/dynamicsprite.cpp:19:
    /usr/include/allegro/alcompat.h:44:7: error: conflicting declaration of C function ‘fixed fadd(fixed, fixed)’
        AL_ALIAS(fixed fadd(fixed x, fixed y), fixadd(x, y))

--- a/debian/README.md
+++ b/debian/README.md
@@ -15,10 +15,12 @@ probably work.
 -   libogg (1.2.2-1.3.0)
 -   libtheora (1.1.1-1.2.0)
 -   libvorbis (1.3.2)
+-   libXext (1.3.3)
+-   libXxf86vm (1.1.4)
 
 Fedora package installation
 ---------------------------
-    yum -y install git allegro-devel dumb-devel freetype-devel libogg-devel libtheora-devel libvorbis-devel
+    yum -y install git allegro-devel dumb-devel freetype-devel libogg-devel libtheora-devel libvorbis-devel libXext-devel libXxf86vm-devel
 
 Debian/Ubuntu package installation
 ----------------------------------

